src/v2/view-builder/views/ov/EnrollPollOktaVerifyView.js

```diff
@@ -36,21 +39,47 @@ const Body = BaseForm.extend(Object.assign(
       this.listenTo(this.model, 'error', this.stopPolling);
       this.startPolling();
     },
+    showMessages () {
+      // override showMessages to display error message
+      const messagesObjs = this.options.appState.get('messages');
+      if (messagesObjs && Array.isArray(messagesObjs.value)) {
+        this.add('<div class="ion-messages-container"></div>', '.o-form-error-container');
+
+        messagesObjs.value.forEach(messagesObj => {
+          const msg = messagesObj.message;
+          if (messagesObj?.class === 'ERROR') {
+            const options = {
+              content: msg,
+              type: 'error',
+            };
+            if (this.options.appState.containsMessageWithI18nKey(OV_FORCE_FIPS_COMPLIANCE_UPGRAGE_KEY_IOS) ||
+            this.options.appState.containsMessageWithI18nKey(OV_FORCE_FIPS_COMPLIANCE_UPGRAGE_KEY_NON_IOS)) {
+              // add a title for ov force upgrade
+              options.title = loc('oie.okta_verify.enroll.force.upgrade.title', 'login');
+            }
+            this.add(createCallout(options), '.o-form-error-container');
+          } else {
+            this.add(`<p>${msg}</p>`, '.ion-messages-container');
+          }
+        });
+
+      }
+    },
     getUISchema () {
       const schema = [];
       const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
-      const selectedChannel = contextualData.selectedChannel;
+      this.selectedChannel = contextualData.selectedChannel;
```

Comment:
why move away from const ?

Reply:
ah, this can be reverted. At first I was going to have keys for email, sms and qr so I wanted the `this.selectedChannel` to be available outside this function.

Reply:
this isnt reverted yet ?

Reply:
Its reverted now